deep-learning/_posts/2017-08-17-S3Pool.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: review
+title: S3Pool - Pooling with stochastic Spatial Sampling
+tags: deep-learning pooling classification
+cite:
+    authors: "S. Zhai, H. Wu, A. Kumar, Y. Cheng, Y. Lu, Z. Zhang, R. Feris "
+    title:   "S3Pool : Pooling with stochastic Spatial Sampling"
+    venue:   "CVPR 2017"
+pdf: "https://arxiv.org/pdf/1611.05138.pdf"
+---
+
+# Idea
+   New kind of pooling aiming to learn from the information that is discarded by classical pooling. The image is spatially partioned and the rows and columns of a pooling grid are non-uniformly and stochastically chosen. It acts as a strong regularizer equivalent to implicit data augmentation because the network will see a different distorsion of the feature maps at each pass.  
+   
+# Method
+1. Partition the image (h,w) into strips of  size g (both vertically and horizontally)
```

Comment:
A few comments on this paragraph:

1. I suggest that you precise that the selected rows/columns are removed (not kept)
2. There is a typo on line 18: siwe -> size
3. I also suggest that you use math display for variables and fractions

Combining those suggestions would give the following:

```
1. Partition the image $$ (h, w) $$ into strips of  size $$ g $$ (both vertically and horizontally)
2. Randomly select $$ \frac{g}{s} $$ rows and columns to be removed from each strip to obtain the downsampling grid
→ the resulting feature map is of size $$ (\frac{h}{s} $$, $$ \frac{w}{s}) $$ (like a pooling with a window of size $$ s $$)
```